cfgov/agreements/jinja2/agreements/_json.html

```diff
@@ -0,0 +1,17 @@
+{# ==========================================================================
+
+   Renders json from a list of credit card agreement issuers.
+
+   issuers: List of credit card agreement issuers.
+
+   ========================================================================== #}
+<div style="display:none" id="issuer-json">
```

Comment:
Actually, I did a third thing in d83ac9b6d82372fbc43e0585c6ee3b8d72c46bfb, included the data directly in a global. This will avoid a JSON.parse at runtime.